compiler/qsc_project/src/project.rs

```diff
@@ -437,7 +437,7 @@ pub trait FileSystemAsync {
             // and open them using cached contents.
             sources.push((
                 format!(
-                    "qsharp-github-source:{}/{}/{}{path}",
+                    "qsharp-github-source:{}/{}/{}/{path}",
```

Comment:
From the integration test failures, it looks like `path` can sometimes have a leading `/` which messes up this string.